File: beginners/ex-5.js
/* Write a function in JavaScript to find and return the maximum element in an array of integers. The function should take an array of integers as input and return the largest value in the array. */

function maxNum(arr) {
    let maxNum = Math.max(...arr);
    return maxNum;
}

// Example usage:
console.log(maxNum([3, 7, 1, 9, 5])); // 9

File: beginners/ex-5.cs
/* Write a function in C# to find and return the maximum element in an array of integers. The function should take an array of integers as input and return the largest value in the array. */

using System;
using System.Linq;

class Program
{
    static int MaxNum(int[] arr)
    {
        int maxNum = arr.Max();
        return maxNum;
    }

    static void Main()
    {
        int[] numbers = { 3, 7, 1, 9, 5 };
        Console.WriteLine(MaxNum(numbers)); // 9
    }
}
